crates/test-programs/tests/wasm_tests/runtime.rs

```diff
@@ -50,6 +50,29 @@ pub fn instantiate(data: &[u8], bin_name: &str, workspace: Option<&Path>) -> any
 
     // The current stable Rust toolchain uses the old `wasi_unstable` ABI,
     // aka `snapshot_0`.
+    module_registry.insert(
+        "wasi_snapshot_preview1".to_owned(),
+        Instance::from_handle(
+            &store,
+            wasmtime_wasi::instantiate_wasi_with_context(
+                global_exports.clone(),
+                builder.build().context("failed to build wasi context")?,
+            )
+            .context("failed to instantiate wasi")?,
+        ),
+    );
```

Comment:
Could we move this comment down, or at least change to wording here, since the code it refers to doesn't seem to be directly below (i.e., the code directly below it seems to instantiate "preview1" instead of  "snapshot0")?

Reply:
Oh oops, yeah this is an outdated comment, I'll fixup